assume/strategies/learning_advanced_orders.py

```diff
@@ -196,6 +196,7 @@ def calculate_bids(
                         "volume": bid_quantity_inflex,
                         "bid_type": "SB",
                         "bid_id": f"{unit.id}_SB_{len(bids) + 1}",
+                        "node": unit.node,
```

Comment:
Is there a particular reason, why you did not add the node to the normal leanring?

Reply:
missed it, thanks